scripts/aws_clear_instance.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+echo "Removing application and dependencies"
+
+if [ -d "/home/notify-app/notifications-api" ]; then
+    # Remove and re-create the directory
+    rm -rf /home/notify-app/notifications-api
+    mkdir -vp /home/notify-app/notifications-api
+    # Remove installed py3 packages
+    pip3 freeze | xargs pip3 uninstall -y
+else
+    echo "Directory does not exist, something went wrong!"
```

Comment:
Although in saying all of this I don't think we should exit here (cause codedeploy to fail). 

If the directory doesn't exist, it could be a new instance. We are using auto-scaling so we do recycle instances but let's say we have a new non-recycled instance, it wouldn't make sense to fail as the directory would never exist. 

I think having the else helps for debugging, in the case where we have a recycled instance and we expect the folder to exist but instead this is printed out.

Feel it should stay as is based on the above.